torch/csrc/jit/serialization/export.cpp

```diff
@@ -1380,16 +1380,36 @@ std::string serialize_model_proto_to_string(
   return model_proto->SerializeAsString();
 }
 
-void check_onnx_proto(const std::string& proto_string, bool full_check) {
+void check_onnx_proto(const std::string& proto_string) {
```

Comment:
This changes public API, no?

Reply:
I think it does.  It's one of TORCH API. Is there anything else I should do?

Reply:
You need to follow https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy
It basically says first emit a warning for 2 releases, then can delete.
But for this function specifically I'm not sure if it's even used by anyone.
We may need to revert if there are complains.

Reply:
Thanks!

Reply:
I've added bc breaking labels.